app/controllers/admin/api/account/proxy_configs_controller.rb

```diff
@@ -27,23 +27,25 @@ class Admin::Api::Account::ProxyConfigsController < Admin::Api::BaseController
   ##~ op.parameters.add @parameter_environment
   ##~ op.parameters.add :name => "host",    :description => "Filter by host",    :dataType => "string", :required => false, :paramType => "query"
   ##~ op.parameters.add :name => "version", :description => "Filter by version", :dataType => "string", :required => false, :paramType => "query"
+  ##~ op.parameters.add @parameter_page
+  ##~ op.parameters.add @parameter_per_page
   #
   def index
-    respond_with proxy_configs.order(:id).paginate(pagination_params)
+    respond_with proxy_configs.order(id: :asc).paginate(pagination_params)
   end
 
   private
 
   def proxy_configs
-    @proxy_configs ||= ProxyConfig.joins(:proxy)
-      .where(proxies: { service_id: accessible_services.pluck(:id) })
-      .by_environment(environment)
-      .by_host(host)
-      .by_version(version)
+    @proxy_configs ||= if version == 'latest'
+                         ProxyConfig.latest_versions(environment: environment, proxy_ids: proxy_ids)
```

Comment:
Same as https://github.com/3scale/porta/pull/2230/files#r531033847. It should be something simpler and more direct here, like `current_account.proxy_configs`, IMO.